app/bundles/PointBundle/Event/GroupEvent.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\PointBundle\Event;
+
+use Mautic\CoreBundle\Event\CommonEvent;
+use Mautic\PointBundle\Entity\Group;
+
+class GroupEvent extends CommonEvent
+{
+    /**
+     * @param bool $isNew
+     */
+    public function __construct(Group &$group, $isNew = false)
+    {
+        $this->entity = &$group;
+        $this->isNew  = $isNew;
```

Comment:
`$isNew` doesn't seem to be used. Is it necessary to pass `$group` as pointer? It is already a pointer for the object type, right?
```suggestion
    public function __construct(private Group $group, $isNew = false)
    {
```

Reply:
Right, I don't think that's necessary. It looked like there was some kind of pattern to creating these kinds of events in Mautic.